### PR TITLE
[agent-smith] Show blacklisted command arguments

### PR DESCRIPTION
**Pull request by aledbf**

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **aledbf** after the PR has been reviewed.

*No associated issue*. Update pull-request body to add a reference to an issue, or get approval with `/approve no-issue`

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[components/ee/agent-smith/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ee/agent-smith/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: components/ee/agent-smith/pkg/agent/agent.go [hunk 1]
```diff
@@ -701,7 +701,10 @@ func (agent *Smith) handleExecveEvent(execve Execve) func() (*InfringingWorkspac
 
 			for _, b := range bl.Binaries {
 				if strings.Contains(execve.Filename, b) || strings.Contains(strings.Join(execve.Argv, "|"), b) {
-					infr := Infringement{Description: fmt.Sprintf("user ran %s blacklisted command: %s", s, execve.Filename), Kind: GradeKind(InfringementExecBlacklistedCmd, s)}
+					infr := Infringement{
+						Description: fmt.Sprintf("user ran %s blacklisted command: %s %v", s, execve.Filename, execve.Envp),
```
Comment (jankeromnes) [inline components/ee/agent-smith/pkg/agent/agent.go:705]: Unfortunately, I don't think `execve.Envp` is ever set:

https://github.com/gitpod-io/gitpod/blob/f646642c05bbd29f520b0df5e9f5fc40bc160318/components/ee/agent-smith/pkg/agent/agent.go#L629-L633

Comment (aledbf) [reply components/ee/agent-smith/pkg/agent/agent.go]: fixed

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: e716328c086fe59a2bbd7b471d87c7f4619a0927</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/5352#issuecomment-905433100" title="Approved">csweichel</a>*

Associated issue requirement bypassed by: *<a href="https://github.com/gitpod-io/gitpod/pull/5352#issuecomment-905433100" title="Approved">csweichel</a>*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[components/ee/agent-smith/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/components/ee/agent-smith/OWNERS)~~ [csweichel]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**